Some tests do not pass

In a brand new environment where `imap-data-access` is installed, some tests fail because they assume that environment variables like `IMAP_API_KEY` and `IMAP_DATA_ACCESS_URL` etc. are not set (which they are for me, as they will be for most developers).

In my case:
```
FAILED tests/test_config_options.py::test_configuration_updates[DATA_DIR-default0-/test/path] - AssertionError: assert '/media/vineetb/T7' == '/media/vinee...a-access/data'
FAILED tests/test_config_options.py::test_configuration_updates[API_KEY-None-test-api-key] - AssertionError: assert 'be24...' == 'None'
```

I think the correct fix is to omit them when calling `subprocess.run` (which inherits all envvars) before asserting that their default value is what the test expects them to be.

--- a/tests/test_config_options.py
+++ b/tests/test_config_options.py
@@ -31,8 +31,14 @@ def test_configuration_updates(config_var, default, expected):
         f"import imap_data_access; print(imap_data_access.config['{config_var}'])",
     ]
     # Default case
+    # Strip any IMAP_ overrides from the current environment so the built-in
+    # default is what gets used in `subprocess.run`.
+    default_env = {
+        key: value for key, value in os.environ.items() if f"IMAP_{config_var}" != key
+    }
     proc = subprocess.run(
         command,
+        env=default_env,
         capture_output=True,
         check=True,
         text=True,

